locales/ara.json

```diff
@@ -258,49 +321,47 @@
     "access": " يطلب الوصول إلى",
     "status": "حالة",
     "successMessage": "لقد قمت بتسجيل الدخول بنجاح ",
-    "okay": "تمام"
+    "okay": "تمام",
+    "allow": "يسمح",
+    "cancel": "يلغي",
+    "essentialClaims": "الادعاءات الأساسية",
+    "voluntaryClaims": "المطالبات الطوعية",
+    "required": "مطلوب"
   },
   "ReceiveVcScreen": {
-    "header": "تفاصيل بطاقة ",
-    "save": "احفظ البطاقة",
-    "acceptRequest": "قبول الطلب واختيار البطاقة",
-    "acceptRequestAndVerify": "قبول الطلب والتحقق",
-    "reject": "رفض",
+    "header": "تفاصيل بطاقة",
+    "save": "احفظ بطاقة",
+    "verifyAndSave": "تحقق وحفظ",
+    "reject": "يرفض",
     "discard": "ينبذ",
-    "goToReceivedVCTab": "عرض البطاقات المستلمة",
-    "saving": "إنقاذ",
+    "goToReceivedVCTab": "عرض تم استلامه بطاقة",
     "errors": {
       "savingFailed": {
         "title": "فشل حفظ بطاقة",
-        "message": "حدث خطأ أثناء حفظ البطاقة في المتجر."
+        "message": "شئ ما ذهب خطأ بينما إنقاذ بطاقة ل ال محل."
       },
       "diskFullError": {
         "title": "فشل حفظ بطاقة",
-        "message": "لا يمكن استلام المزيد من بطاقةالبطاقات أو حفظه لأن بيانات التطبيق ممتلئة."
+        "message": "لا يمكن استلام المزيد من {{vcLabelPlural}} أو حفظه لأن بيانات التطبيق ممتلئة."
```

Comment:
We have replaced the {{vc label}} place holders with direct native translations, please change it accordingly.

Reply:
Translated to the required word in the respective Arabic language.